### PR TITLE
Edit new_resource partial

### LINKED ISSUE #4109: Missing translations on new delivery method setup page
## Description

URL: `/admin/shipping_methods/new`

Capture du 2019-07-31 22-15-16

Also, when I change the calculator type, a little sentence appear below, that is untranslated :
Capture du 2019-07-31 22-39-23

### PR DESCRIPTION
**Pull request by myersca1**
#### What? Why?

Closes #4109 

New_resource_links file was referenced in multiple places, but did not exist. The existing .erb file did not work, so a .haml file was created in its place. 

#### What should we test?
Verify that all pages referencing this file work properly with the translations.

Changelog Category: Fixed

### CODE REVIEW

File: app/views/spree/admin/shared/_new_resource_links.html.haml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.form-buttons.filter-actions.actions
+  = button t(:create), 'icon-ok'
+  %span.or= t(:or)
+  = button_link_to t(:cancel), collection_url, icon: 'icon-remove'
```
Comment (luisramos0) [inline app/views/spree/admin/shared/_new_resource_links.html.haml]: I wonder why the change from link_to_with_icon to button_link_to. What is the differrence between these two methods? Will the user notice any difference?

File: app/views/spree/admin/shared/_new_resource_links.html.haml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.form-buttons.filter-actions.actions
+  = button t(:create), 'icon-ok'
```
Comment (luisramos0) [inline app/views/spree/admin/shared/_new_resource_links.html.haml]: Looking at the code now, to fix the issue, all you needed would be to change the translation calls from Spree.t to t. because spree.actions.create and spree.actions.cancel are not there in the en.yml file but both are under actions.create (Spree.t is similar to t. but adds spree in the namespace).

Here you are moving to root level :create and :cancel. I think it's ok, I would prefer to keep these under the actions namespace, so I'd use t('actions.create') and t('actions.cancel') instead of t(:create) and t(:cancel).
The OR can stay as t(:or).

File: app/views/spree/admin/shared/_new_resource_links.html.haml (hunk 1 above)
Comment (blainebillings) [reply app/views/spree/admin/shared/_new_resource_links.html.haml]: The only difference is purely aesthetic - whether or not it looks like a button, as far as I could tell, but maybe I am mistaken.

File: app/views/spree/admin/shared/_new_resource_links.html.haml (hunk 2 above)
Comment (blainebillings) [reply app/views/spree/admin/shared/_new_resource_links.html.haml]: Thank you very much - that makes complete sense. We went ahead and updated with the change.

**Comment by RachL** [comment]
hehe even already on transifex 💪 

image


This is ready to go!

**Comment by luisramos0** [comment]
Congrats @myersca1 🎉

**Comment by RachL** [comment]
ah yes welcome @myersca1 !

Where are my manners 🙈
